contracts/FlowCallbackScheduler.cdc

```diff
@@ -383,6 +390,10 @@ access(all) contract FlowCallbackScheduler {
                 executionEffort: executionEffort
             )
 
+            if scheduledTimestamp == nil {
```

Comment:
This should only happen if priority is high, for medium and low priority we always scheduled a timestamp.